lib/api/voice.ts

```
@@ -0,0 +1,52 @@
+    
+    const response = await authenticatedApiClient.get(`/chat/voice/tts/${cid}`, {
+      responseType: 'blob',
+    });
+
+    // console.log('VoiceService.getTtsAudio 응답:', { 
+    //   status: response.status, 
+    //   blobSize: response.data.size 
+    // });
+    return response.data;
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)